cmd/vic-machine-server/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	loads "github.com/go-openapi/loads"
+	flags "github.com/jessevdk/go-flags"
+
+	"github.com/vmware/vic/lib/apiservers/service/restapi"
+	"github.com/vmware/vic/lib/apiservers/service/restapi/operations"
+)
+
+// This file was generated by the swagger tool.
+// Make sure not to overwrite this file after you generated it because all your edits would be lost!
+
+func main() {
+
```

Comment:
extra line

Reply:
> This file was generated by the swagger tool.

Given that the file was generated, and `go fmt` didn't remove it, I'm inclined to leave it as-is.

If I start touching the file, I'm going to wonder if I should also remove line 55, wonder why there's no line between 32 and 33, but is a line between 50 and 52, and... none of that seems like a good use of time.